.buildkite/script/run_linter_check.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# !!! WARNING DO NOT add -x to avoid leaking vault passwords
+set -euo pipefail
+
+source .buildkite/script/shared.sh
+
+if is_pr && ! is_fork && ! has_skip_label; then
+  echo "We're on PR, running autoformat"
+
+  export GH_TOKEN="$VAULT_GITHUB_TOKEN"
+  source .buildkite/script/git-setup.sh
+
+  if ! make autoformat ; then
+    echo "make autoformat ran with errors, exiting"
+    exit 1
+  fi
+
+  if [ -z "$(git status --porcelain)" ]; then
+    echo "Nothing to be fixed by autoformat"
+    exit 0
+  else
+    git --no-pager diff
+    echo "linting errors are fixed, pushing the diff"
+
+    git add .
+    git commit -m"make autoformat"
```

Comment:
```suggestion
    git commit -m "make autoformat"
```
I don't _think_ this matters 🤷

Reply:
I always do without a space :)